cluster/gce/upgrade.sh

```diff
@@ -76,6 +76,9 @@ function upgrade-master() {
   detect-master
   parse-master-env
 
+  MASTER_OS_DISTRIBUTION=$(get-node-os "${MASTER_NAME}")
```

Comment:
You could add it as a flag maybe or check if the env variable is already defined.